athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneConnection.java

```diff
@@ -21,21 +21,27 @@
 
 import org.apache.tinkerpop.gremlin.driver.Client;
 import org.apache.tinkerpop.gremlin.driver.Cluster;
+import org.apache.tinkerpop.gremlin.driver.SigV4WebSocketChannelizer;
 import org.apache.tinkerpop.gremlin.driver.remote.DriverRemoteConnection;
 import org.apache.tinkerpop.gremlin.process.traversal.AnonymousTraversalSource;
 import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
 
 public class NeptuneConnection
 {
-     private static Cluster cluster = null;
+    private static Cluster cluster = null;
 
-    NeptuneConnection(String neptuneEndpoint, String neptunePort)
+    NeptuneConnection(String neptuneEndpoint, String neptunePort, Boolean enabledIAM)
```

Comment:
Why Boolean (boxed type) as opposed to Boolean (non-boxed)?

Reply:
You may need to add a null check somewhere in order to use the non-boxed form but I think that is preferred for how you are using this as null seems to be akin to false for this usage.

Reply:
can convert this to primitive type boolean.  yes, the default value is false, even if user doesn't supply a value. Boolean.parseBoolean(System.getenv("iam_enabled")) will return false.